src/components/Dialog/HotkeySettingDialog.vue

```diff
@@ -298,26 +335,98 @@ const changeHotkeySettings = (
   });
 };
 
-const duplicatedHotkey = computed(() => {
-  if (lastRecord.value == "") return undefined;
-  return hotkeySettings.value.find(
-    (item) =>
-      item.combination == lastRecord.value && item.action != lastAction.value
+const duplicatedHotkeys = computed(() => {
+  return filterDuplicatedHotkeysWithSetKeyAndName(
+    lastRecord.value,
+    lastAction.value as HotkeyActionNameType
   );
 });
 
+const filterDuplicatedHotkeysWithSetKeyAndName = (
+  key: HotkeyCombination,
+  action: HotkeyActionNameType
+): HotkeySettingType[] => {
+  if (key == "") return [];
+  if (lastArgumentKeyText.value == "") {
+    const result = findDuplicatedHotkeyWithKey(key, action);
+    if (result == undefined) {
+      return [];
+    } else {
+      return [result];
+    }
+  } else {
+    const editingHotkey = getEditingHotkey(action);
+    const argumentKeys = getArgumentKeyCombination(editingHotkey.argumentKey);
+    const results: HotkeySettingType[] = [];
+    for (const argumentKey of argumentKeys) {
+      lastRecord.value + " " + argumentKey;
+      const result: HotkeySettingType | undefined = findDuplicatedHotkeyWithKey(
+        (lastRecord.value + " " + argumentKey) as HotkeyCombination,
+        lastAction.value as HotkeyActionNameType
```

Comment:
`HotkeyActionNameType`は`as`を使うしかないのですが、`HotkeyCombination`の方は関数化されてるのでこう書いたほうが分かりやすいかなと！
```suggestion
        HotkeyCombination(lastRecord.value + " " + argumentKey),
        lastAction.value as HotkeyActionNameType
```